Docs/source/inputs.rst

```diff
@@ -2,70 +2,41 @@
 Input Files
 ***********
 
-The Castro executable uses two inputs files at runtime to set and
-alter the behavior of the algorithm and initial conditions:
+The Castro executable uses an inputs file at runtime to set and
+alter the behavior of the algorithm and initial conditions.  Typically
+named ``inputs``, it 
 
-  * The main inputs file, typically named ``inputs``:
+  * Sets the AMReX parameters for gridding, refinement, etc., through the
+    ``geometry`` and ``amr`` namespaces.
 
-    This is used to set AMReX parameters and the control flow in the
-    physics solvers in Castro.  Each parameter here has a namespace
-    (like ``amr.optionname`` or ``castro.optionname``).  Parameters
-    set here are read using the AMReX ``ParmParse`` class
-    infrastructure.
+  * Enables different physics behaviors through the ``castro`` namespace.
 
-    .. warning:: Because the inputs file is handled by the C++ portion
-       of the code, any quantities you specify in scientific notation,
-       must take the form ``1.e5`` and not ``1.d5``—the ``d``
-       specifier is not recognized.
+  * Sets the problem-specific runtime parameters through the ``problem`` namespace.
 
-  * The problem / external inputs file, typically named ``probin``:
+  * Sets any Microphysics runtime parameters, through the various namespaces
+    defined in Microphysics (like ``eos``, ``integrator``, ``network``, ...).
 
-    This is used by the Fortran code that initializes the problem
-    setup.  It is read at problem initialization (via a Fortran
-    namelist) and the problem-specific quantities are stored in a
-    Fortran module ``probdata_module`` defined in the problem’s
-    ``probdata.f90`` file.
+.. warning:: Because the inputs file is handled by the C++ portion
+   of the code, any quantities you specify in scientific notation,
+   must take the form ``1.e5`` and not ``1.d5``—the ``d``
+   specifier is not recognized.
 
-    The ``probin`` file also has an ``&extern`` namelist that is
-    used to specify runtime parameters for the microphysics
-    (EOS, reaction network, ...)
 
-Only the ``inputs`` file is specified on the commandline. The
-associated ``probin`` file is specified in the inputs file
-using the ``amr.probin_file`` parameter, e.g.::
-
-    amr.probin_file = my_special_probin
-
-for example, has the Fortran code read a file called ``my_special_probin``.
+Only the ``inputs`` file is specified on the commandline.
```

Comment:
I think this sentence looks a little odd without the context about probin *not* being set at the command line, maybe just delete it?

Reply:
right, I think the point I want to remind is that we need to set an inputs file on the commandline

Reply:
That would be fine, maybe just remove the "only" then.

Reply:
well... I guess you can do it all as runtime parameters.  I'll remove the line.